openedx/adg/lms/applications/views.py

```diff
@@ -1,3 +1,39 @@
 """
 All views for applications app
 """
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import HttpResponseRedirect
+from django.urls import reverse_lazy
+from django.views.generic.edit import FormView
+
+from openedx.adg.lms.applications.forms import ContactInformationForm
+
+
+class ContactInformationView(LoginRequiredMixin, FormView):
+    """
+    View for the contact information of user application
+    """
+
+    template_name = 'adg/lms/applications/contact_info.html'
+    form_class = ContactInformationForm
+    login_url = '/register'
+    success_url = reverse_lazy('application_experience')
+
+    def form_valid(self, form):
+        form.save(request=self.request)
+        if form.cleaned_data.get('resume'):
+            return HttpResponseRedirect(reverse_lazy('application_cover_letter'))
```

Comment:
Is it requirement ? because I couldn't see it in AC of the ticket

Reply:
Its mentioned in the figma designs